buckw

```diff
@@ -129,8 +129,12 @@ runOkBuck ( ) {
 
     if [[ ! -z "$INSTALLED_WATCHMAN" ]]; then
         getToClean | jsonq '"\n".join(obj["files"])' | xargs rm
-        info "DELETED OLD BUCK FILES"
-        EXTRA_OKBUCK_ARGS="-xokbuckClean $EXTRA_OKBUCK_ARGS"
+        if [[ $? -eq 0 ]]; then
+            info "DELETED OLD BUCK FILES"
+            EXTRA_OKBUCK_ARGS="-xokbuckClean $EXTRA_OKBUCK_ARGS"
+        else
+            info "Watchman Failed relying on okbuckClean"
```

Comment:
Lets change this to warn

make it all caps

`Watchman Failed. Using okbuckClean`